readme.md

```diff
@@ -9,6 +9,22 @@ This repository contains the following:
 - **Microsoft Azure Provisioning device SDK for C#** to provision devices to Azure IoT Hub with .NET.
 - **Microsoft Azure Provisioning service SDK for C#** to manage your Provisioning service instance from a back-end .NET application.
 
+## Critical Upcoming Change Notice
```

Comment:
Might be good to add a link to the LTS release notes once it is released or the version.